_data/tools.yml

```diff
@@ -206,6 +206,14 @@
   v2: true
   v3: true
 
+- name: Bump
+  category: documentation
+  link: https://bump.sh
+  language: SaaS
+  description: Bump generates elegant documentations and automatic changelogs from your Swagger/OpenAPI specifications. Git diff, for your API.
```

Comment:
Can we remove Swagger from this description? They changed their name a long time ago, and at some point we need to just accept that its not the name anymore.

Reply:
Yes, you're totally right. Change made.